move-mutation-test/src/cli.rs

```diff
@@ -0,0 +1,204 @@
+// Copyright © Eiger
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::too_long_first_doc_paragraph)]
+
+use aptos::common::types::{MovePackageDir, OptimizationLevel};
+use aptos_framework::extended_checks;
+use clap::Parser;
+use move_compiler_v2::Experiment;
+use move_model::metadata::LanguageVersion;
+use move_mutator::cli::{FunctionFilter, ModuleFilter};
+use move_package::CompilerConfig;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+/// Command line options for mutation test tool.
+#[derive(Parser, Default, Debug, Clone, Deserialize, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct CLIOptions {
+    /// Work only over specified modules.
+    #[clap(long, value_parser, default_value = "all")]
+    pub mutate_modules: ModuleFilter,
+
+    /// Work only over specified functions.
```

Comment:
Are these qualified functions? (i.e., `mod_a::function_a` vs. `function_a`)

Reply:
These are not qualifed functions. I will update the comment above.

To test e.g. `mod_a::function_a`, the user should specify: `function_a` for the `mutate_functions` parameter and `mod_a` within the `mutate_modules` parameter